Fluid.MvcSampleVB/Program.vb

```diff
@@ -4,7 +4,7 @@ Imports Microsoft.AspNetCore.Hosting
 
 Module Program
     Sub Main(args As String())
-        Dim host = New WebHostBuilder().
+        Dim host As IWebHost = New WebHostBuilder().
```

Comment:
It's enough to use
```vb
Dim host As New WebHostBuilder()
```

Reply:
If it is done like that I would then be unable to call `host.Run()` as it belongs to `IWebHost`?

Reply:
I see, so that's why I surprised how `New WebHostBuilder().` it works with you without using `Call` 😄 

thanks

Reply:
ahh I see you are calling it directly inline as one. I misread your example earlier 😊

Reply:
No problem even your latest suggestion was fine
thanks again